lib/cfg.c

```diff
@@ -447,6 +447,10 @@ cfg_new(gint version)
   self->use_uniqid = FALSE;
   self->jvm_options = NULL;
 
+  self->logstore_journal_shmem_threshold = 536870912;
```

Comment:
We could write 512 * 1024 *1024. It would be a little more cleaner.

Reply:
done